src/Foundation/Commerce/code/Services/Account/AccountService.cs

```diff
@@ -16,60 +16,387 @@ namespace Wooli.Foundation.Commerce.Services.Account
 {
     using System;
     using System.Collections.Generic;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+    using System.Web.Security;
 
-    using Base.Models;
     using Base.Models.Result;
 
+    using Connect.Context.Storefront;
+    using Connect.Managers.Account;
+
     using DependencyInjection;
 
+    using Mappers.Account;
+
     using Models.Entities.Addresses;
     using Models.Entities.Users;
 
+    using Sitecore.Commerce.Engine.Connect.Entities;
+    using Sitecore.Commerce.Entities;
+    using Sitecore.Commerce.Entities.Customers;
+    using Sitecore.Diagnostics;
+
     [Service(typeof(IAccountService), Lifetime = Lifetime.Singleton)]
     public class AccountService : IAccountService
     {
+        private readonly IAccountManagerV2 accountManager;
+
+        private readonly IAccountMapper mapper;
+
+        private readonly IStorefrontContext storefrontContext;
+
+        public AccountService(
+            IAccountManagerV2 accountManager,
+            IAccountMapper accountMapper,
+            IStorefrontContext storefrontContext)
+        {
+            Assert.ArgumentNotNull(accountManager, nameof(accountManager));
+            Assert.ArgumentNotNull(accountMapper, nameof(accountMapper));
+            Assert.ArgumentNotNull(storefrontContext, nameof(storefrontContext));
+
+            this.accountManager = accountManager;
+            this.mapper = accountMapper;
+            this.storefrontContext = storefrontContext;
+        }
+
         public Result<IEnumerable<Address>> AddAddress(string userName, Address address)
```

Comment:
Why does add address return value? What's the difference between adding and removing or updating?